Core/include/Acts/Seeding/Seedfinder.hpp

```diff
@@ -41,10 +52,31 @@ class Seedfinder {
     // ...for middle-top
     std::vector<LinCircle> linCircleTop;
 
+    // parameters for central seed confirmation
+    SeedConfirmationRange centralSeedConfirmationRange = {
```

Comment:
Just looking at this part of the code, I see you are putting the ranges in the header of the `Seedfinder`. The best would be to include the definition of this struct and the two confirmation ranges for central and forward to the config so that they can be configured.